spec/Accessport2LtpTranslator+services.yaml

```diff
@@ -1,262 +1,217 @@
       individual:
-        - service-name: /v1/bequeath-your-data-and-die
-          uuid: xx-1-0-0-op-s-im-000
+        - operation-name: /v1/bequeath-your-data-and-die
+          uuid: a2lt-1-0-0-op-s-im-000
     service:
       basic:
-        - service-name: /v1/inform-about-application
-          uuid: xx-1-0-0-op-s-bs-001
-        - service-name: /v1/inform-about-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-002
-        - service-name: /v1/inform-about-release-history
-          uuid: xx-1-0-0-op-s-bs-003
-        - service-name: /v1/inform-about-release-history-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-004
+        - operation-name: /v1/start-application-in-generic-representation
+          uuid: a2lt-1-0-0-op-s-bs-000
+        - operation-name: /v1/inform-about-application
+          uuid: a2lt-1-0-0-op-s-bs-001
+        - operation-name: /v1/inform-about-application-in-generic-representation
+          uuid: a2lt-1-0-0-op-s-bs-002
+        - operation-name: /v1/inform-about-release-history
+          uuid: a2lt-1-0-0-op-s-bs-003
+        - operation-name: /v1/inform-about-release-history-in-generic-representation
+          uuid: a2lt-1-0-0-op-s-bs-004
+        - operation-name: /docs
+          uuid: a2lt-1-0-0-op-s-bs-005
       individual:
-        - service-name: /v1/start-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-is-000
-        ## TODO: Add Services specific to this application
+        - service-name: /v1/translate-equipment-to-ltp
+          uuid: a2lt-1-0-0-op-s-is-000
   http-server:
-    own-application-name: ## TODO: Add OwnApplicationName
-    own-release-number: ## TODO: Add OwnReleaseNumber (e.g. 1.0.0)
-    uuid: xx-1-0-0-http-s-bm-000
+    own-application-name: Accessport2LtpTranslator
+    own-release-number: 1.0.0
+    uuid: a2lt-1-0-0-http-s-000 ## before: a2lt-1-0-0-http-s-bm-000
   tcp-server:
-    own-ip-address: ## TODO: Add OwnIpAddress
-    own-tcp-port: ## TODO: Add OwnTcpPort
-    uuid: xx-1-0-0-tcp-s-bm-000
-
+    - description: Without TLS layer ##Transport Layer Security
+      local-protocol: http
+      own-ip-address: 1.1.4.3
+      own-tcp-port: 4003
+      uuid: a2lt-1-0-0-tcp-s-000
+    - description: With TLS layer ##Transport Layer Security
+      local-protocol: https
+      own-ip-address: 1.2.4.3 ## from https://github.com/openBackhaul/ApplicationPattern/blob/develop/doc/TestingApplications/Infrastructure/SdnLaboratory/FakeAddresses/IpAddresses.md
+      own-tcp-port: 4203
+      uuid: a2lt-1-0-0-tcp-s-001
 
 
 clients:
 
+  - http-client:
+      application-name: ElasticSearch
+      release-number: 1.0.0
+      uuid: a2lt-1-0-0-http-c-es-1-0-0-000
+    tcp-client:
+      remote-protocol: http
+      ip-address: 1.1.3.15
+      tcp-port: 3015
+      uuid: a2lt-1-0-0-tcp-c-es-1-0-0-000
+    elasticsearch-client:
+      uuid: a2lt-1-0-0-es-c-es-1-0-0-000
+      index-alias: ## TODO: Add IndexAlias
+
```

Comment:
den ElasticSearch client Block brauchst du nur, falls du auch Daten in der ElasticSearch-Datenbank speichern willst
(wenn das sehr viele Daten sind, wie beim MWDI sollte man das machen;
falls du gar keine Daten speichern willst, kann das weg. Oder wenn du Daten speichern willst, das aber nur wenige Daten sind, dann sollen die in einem File gespeichert werden (dafür müsste dann bei den ProfileInstances eine File-Instanz eingetragen werden)

Reply:
Es könnte Sinn machen Responses des Services zu cachen.
Ob das Sinn macht kann ich nicht beurteilen.
Für eine Speicherung von Daten in Files sehe ich keine Anwendung.

Reply:
Die Frage ist in dem Fall einfach, wie groß die Antworten sind. Wenn das riesige Datenmengen sind, wie beim MWDI sollten die in der Datenbank gespeichert werden. 
Wenn das aber nur eine kleine Datenmenge ist, dann würde der Cache intern in der App über eine Datei abgebildet (das wäre nach außen gekapselt, der User/andere Apps wissen gar nicht, dass der Cache über eine Datei läuft).

So hab ich das zumindestens verstanden... Wir sollten Thorsten im nächsten AppOwner-Meeting nochmal fragen.